tests/testthat/test-count_entities.R

```diff
@@ -128,10 +128,9 @@ test_that("counting the number of strongly connected components is possible", {
      create_graph() %>%
      add_islands_graph(
        n_islands = 4,
-       island_size = 10,
-       p = 1 / 5,
-       edges_between = 1,
-       set_seed = 23)
+       island_size = 1,
```

Comment:
with this if I follow correctly we created 4 unconnected islands of 1 elements each so there are 4 strongly connected elements. however are they connected to themselves then?

Reply:
I think we should go with this for now and solve the immediate problem (of fixing a revdep failure for igraph). I can always revise the tests later.